pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.igniterealtime.openfire.botz</groupId>
     <artifactId>botz</artifactId>
-    <version>1.2.1-SNAPSHOT</version>
+    <version>1.2.2-SNAPSHOT</version>
```

Comment:
This is quite a major change. I suggest bumping the version number to 1.3 instead of 1.2.2!